SonarQube.TeamBuild.PreProcessor/Roslyn/EmbeddedAnalyzerInstaller.cs

```diff
@@ -105,54 +120,75 @@ public IEnumerable<string> InstallAssemblies(IEnumerable<Plugin> plugins)
         /// </summary>
         private static string GetLocalCacheDirectory()
         {
-            string localCache = Path.Combine(Path.GetTempPath(), ".sq", ".static");
+            string localCache = Path.Combine(Path.GetTempPath(), ".sonarqube", ".static");
             return localCache;
         }
         
         private IEnumerable<string> GetPluginResourceFiles(Plugin plugin)
         {
+            this.logger.LogDebug(RoslynResources.EAI_ProcessingPlugin, plugin.Key, plugin.Version);
+
             string cacheDir = GetResourceSpecificDir(this.localCacheDirectory, plugin);
 
-            if (Directory.Exists(cacheDir))
+            IEnumerable<string> allFiles = FetchFilesFromCache(cacheDir);
```

Comment:
Previously we'd treat finding the expected directory as a cache hit, even if it was empty (e.g. because we couldn't download the files). Now empty directories are treated as cache misses.